### PR TITLE
Update on 05-linear-regression-simple.md

### PR DESCRIPTION
**Pull request by ashwin3005**

### CODE REVIEW

File: course-zoomcamp/02-regression/05-linear-regression-simple.md [hunk 1]
```diff
@@ -13,7 +13,21 @@ Model for solving regression tasks, in which the objective is to adjust a line f
 So the simple linear regression formula looks like: 
 $g(x_i) = w_0 + x_{i1} \cdot w_1 + x_{i2} \cdot w_2 + ... + x_{in} \cdot w_n$.
 
-And that can be further simplify: $g(x_i) = w_0 + \displaystyle\sum_{j=1}^{n} w_j \cdot x_{ij}$
+And that can be further simplify: $g(x_i) = w_0 + \displaystyle\sum_{j=1}^{n} w_j \cdot x_{ij}$\
+Here is a simple implementation of Linear Regression in python:\
```
Comment (alexeygrigorev) [inline course-zoomcamp/02-regression/05-linear-regression-simple.md]: instead of the slash can you add a few linebreaks?

Comment (ashwin3005) [reply course-zoomcamp/02-regression/05-linear-regression-simple.md]: sure..

File: course-zoomcamp/02-regression/05-linear-regression-simple.md [hunk 2]
```diff
@@ -13,7 +13,21 @@ Model for solving regression tasks, in which the objective is to adjust a line f
 So the simple linear regression formula looks like: 
 $g(x_i) = w_0 + x_{i1} \cdot w_1 + x_{i2} \cdot w_2 + ... + x_{in} \cdot w_n$.
 
-And that can be further simplify: $g(x_i) = w_0 + \displaystyle\sum_{j=1}^{n} w_j \cdot x_{ij}$
+And that can be further simplify: $g(x_i) = w_0 + \displaystyle\sum_{j=1}^{n} w_j \cdot x_{ij}$<br>
```
Comment (alexeygrigorev) [inline course-zoomcamp/02-regression/05-linear-regression-simple.md]: Not with br

Just add a linebreak like here

Line 1

Line 2 (with two linebreaks between Line 1)

Comment (alexeygrigorev) [reply course-zoomcamp/02-regression/05-linear-regression-simple.md]: So basically remove br and hit enter two times =)

Comment (ashwin3005) [reply course-zoomcamp/02-regression/05-linear-regression-simple.md]: okay, I will do that.

**Pull request merged**